server/build.gradle.kts

```diff
@@ -76,7 +76,25 @@ tasks.register<GenerateTask>("openapiGenerateDeltaSharing") {
 
 tasks.withType<Test> {
```

Comment:
In this version, only "normal" tests will run. no test report for delta tests is generated (i.e. they are not run).
I will push the changes needed to do so, have a look :)

Anyway, good job ;)